Dockerfile

```diff
@@ -25,6 +26,7 @@ RUN CGO_ENABLED=1 \
   GOOS=linux \
   go build \
   -tags "${BUILD_TAGS}" \
+  -ldflags="-X github.com/pocket-id/pocket-id/backend/internal/common.Version=${VERSION}" \
```

Comment:
I've added this so the version is stamped in the binary, which lets code retrieve the value.

It can be set at build time using `docker build --build-arg VERSION=$(cat .version)` for example.